addons/event/tests/test_event_flow.py

```diff
@@ -136,3 +136,18 @@ def test_event_date_range(self):
         self.assertEqual(self.event_0.registration_ids.get_date_range_str(), u'next month')
 
         self.patcher.stop()
+
+    def test_event_is_registrable(self):
```

Comment:
please, write your test un even_Sale, and add use case whith ticket, and product_id archived.

If your test was green before, when you call super and do nothing with it, it is the proof that you test don't cover all the cases